doc/source/serve/production-guide/kubernetes.md

```diff
@@ -28,18 +28,25 @@ Follow the [KubeRay quickstart guide](kuberay-quickstart) to:
 * Prepare a Kubernetes cluster
 * Deploy a KubeRay operator
 
-(serve-deploy-app-on-kuberay)=
-## Deploying a Serve application
+## RayService CR
+Once the KubeRay controller is running, you can manage your Ray Serve application by creating and updating a `RayService` custom resource (CR, here is [example](https://github.com/ray-project/kuberay/blob/release-0.5/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml)).
+Under the `spec` section in the `RayService` CR, you can find the following fields:
+
+**`serviceUnhealthySecondThreshold`**: represents the threshold in seconds that defines when a service is considered unhealthy. Default is 60 seconds. When the service is unhealthy, the KubeRay Service controller will try to recreate a new cluster and deploy the application to the new cluster.
+
+**`deploymentUnhealthySecondThreshold`**: represents the threshold in seconds that defines when the ray dashboard is considered unhealthy. Default is 60 seconds. When the ray dashboard is unhealthy, the KubeRay Service controller will try to recreate a new cluster and deploy the application to the new cluster.
```

Comment:
Based on [this note](https://github.com/shrekris-anyscale/kuberay/blob/af7f80b4b09a01559744ce7f0f262c94242a9be6/docs/guidance/rayservice.md?plain=1#L126), I think `deploymentUnhealthySecondThreshold` limits how long a Serve config should take to be deployed. I don't think it's about the Ray dashboard's health. Could you add more details here?